### ISSUE TITLE
Unexpected Trace shutdown output in CSV downloads from EDD

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-bwtrace#41]
The bw_trace_ok_to_echo() function does not detect the fact that a CSV file is being downloaded in EDD. The CSV file therefore gets corrupted. We need a mechanism to prevent this from happening.
There's probably an action hook we can detect.

**Comment by bobbingwide** [comment]
Hasn’t this been fixed in #78

**Comment by bobbingwide** [comment]
The request for an EDD Export is /wp-admin/edit.php?post_type=download&page=edd-tools&tab=import_export
with the following values:
```
_REQUEST Array
(
    [post_type] => download
    [page] => edd-tools
    [tab] => import_export
    [edd_action] => export_settings
    [edd_export_nonce] => 73a8dcbe83
    [_wp_http_referer] => /wp-admin/edit.php?post_type=download&page=edd-tools&tab=import_export
    [submit] => Export
)
```

So we can just check for `edd_action`
